app/src/main/java/org/simple/clinic/summary/PatientSummaryEffectHandler.kt

```diff
@@ -96,7 +96,9 @@ class PatientSummaryEffectHandler @AssistedInject constructor(
           .observeOn(schedulersProvider.io())
           .map {
             val currentFacilityId = currentFacility.get().uuid
-            CDSSPilotStatusChecked(isPilotEnabledForFacility = cdssPilotFacilities.get().contains(currentFacilityId))
+            CDSSPilotStatusChecked(isPilotEnabledForFacility =
```

Comment:
shouldn't this parameter just be named `isCDSEnabledForFacility` now? If CDS is enabled for all facilities, there shouldn't be any need for pilots anymore.

Reply:
Also, indentation seems off.

Reply:
@qptr This is not enabled for all facilities yet. Only enabled in ethiopia and selected facilities in other countries. Also, the indention is default for kotlin language and get set automatically.

Reply:
Oh, does pilot mean CDS feature here?

Reply:
yes